ios/RNMarigoldBridge.m

```diff
@@ -35,8 +35,8 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge {
 }
 
 - (NSArray<id<RCTBridgeModule>> *)extraModulesForBridge:(RCTBridge *)bridge {
-    RNSailthruMobile *rnSailthruMobile = [[RNSailthruMobile alloc] initWithDisplayInAppNotifications:self.displayInAppNotifications];
-    NSArray *modules = @[ rnSailthruMobile ];
+    RNMarigold *rnMarigold = [[RNMarigold alloc] initWithDisplayInAppNotifications:self.displayInAppNotifications];
```

Comment:
We need to add the `RNEngageBySailthru` class here to add it to the list of modules for customers not using auto linking.